.circleci/config.yml

```diff
@@ -100,6 +101,37 @@ jobs:
             - target/debug/deps
           key: cargocache-v2-package_std-rust:1.41.1-{{ checksum "Cargo.lock" }}
 
+  package_storage:
+    docker:
+      - image: rust:1.41.1
+    steps:
+      - checkout
+      - run:
+          name: Version information
+          command: rustc --version; cargo --version; rustup --version; rustup target list --installed
+      - restore_cache:
+          keys:
+            - cargocache-v2-package_storage-rust:1.41.1-{{ checksum "Cargo.lock" }}
+      - run:
+          name: Build library for native target
+          working_directory: ~/project/packages/storage
+          command: cargo build --locked
```

Comment:
I think this should also have a wasm build just like cosmwasm_std

Reply:
I can, but there is no wasm-dependent code in this package, unlike cosmwasm-std, which has wasm target flags. It just depends on the generic traits from cosmwasm-std.

Reply:
👍 yeah, right, not even an `usize` used